start-using-polymer/readme.md

```diff
@@ -10,29 +14,38 @@ The idea 💡is that you write elements that are custom made for one goal. They
 
-### There are three distinct 'languages' in Polymer
+### There are three distinct 'languages' in web development so, also in  Polymer
 
-A polymer element consists of three languages: [html](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Introduction), [css](https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Getting_started) and [javascript](https://developer.mozilla.org/en-US/Learn/JavaScript).  
-HTML is for structure, css is for styling and javascript is for logic. Each one has it's own language and rules. The order we use is first css, then html and js at the end. ```<Style>``` indicates CSS and ```<script>``` is javascript.
-> Note that some standard implementations of html/css/javascript are not valid in Polymer ❗️ This makes finding (valid) help online somewhat more difficult. Polymer has been updated as well, rendering some answers to earlier questions false 🚫. Use the filter date function on Google to limit search results to +-4 months.
+A polymer element consists of three languages: [HTML](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Introduction), [CSS](https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Getting_started) and [javascript](https://developer.mozilla.org/en-US/Learn/JavaScript).  
+HTML is for structure, CSS is for styling and javascript is for logic. Each one has it's own language and rules. Each polymer document is a ```.html``` file. The order we use in each polymer document begins with HTML base elements, within this base we write first CSS, then  we build up the HTML (consists of polymer elements) and javascript at the end. ```<Style>``` indicates CSS and ```<script>``` is javascript.
+
+> Note that some standard implementations of HTML/CSS/javascript are not valid in Polymer ❗️ This makes finding (valid) help online somewhat more difficult. Polymer has been updated as well, [rendering](../glossary/rendering.md) some answers to earlier questions false 🚫. Use the filter date function on Google to limit search results to +-4 months.
 
 ### So now what...?
 
-Read all our other [guides on Polymer](https://github.com/newatoms/guides) and start implementing something. Since we're using Git you can't really destroy anything except for the database. Please don't do that 😬. See our [create a new element guide](https://github.com/newatoms/newatoms/blob/ready/internal/guides/how-to-create-a-new-element.md) on how to avoid this. Ask a team member if you're stuck or if you're afraid you'd break 🔥 something. Good luck! 😉
+Read all our other [guides on Polymer](https://hub.com/newatoms/guides) and start implementing something. Since we're using [Git](../glossary/git.md) you can't really destroy anything except for the database. Please don't do that 😬. See our [create a new element guide](https://github.com/newatoms/newatoms/blob/ready/internal/guides/how-to-create-a-new-element.md) on how to avoid this. Ask a team member if you're stuck or if you're afraid you'd break 🔥 something. Good luck! 😉
```

Comment:
Yeah you're kinda right. It was an old link when the guides were in the repo repository, haven't checked it 😋. Is it better to say: *You can find [all our other guides on polymer in the index](../readme.md), which will give you the information you need to start implementing something.* If I make separate links for all the guides they might be quickly outdated, because I can imagine they will increase quite rapidly. I can also just *not* link to the index.